.github/workflows/ci.yml

```diff
@@ -23,14 +23,14 @@ jobs:
       - name: Use Node.js ${{ env.NODE_VERSION }}
         uses: actions/setup-node@v2
         with:
-          cache: 'yarn'
+          cache: 'npm'
           node-version: ${{ env.NODE_VERSION }}
 
       - name: Install dependencies
-        run: yarn install --frozen-lockfile
+        run: npm install --frozen-lockfile
```

Comment:
in fact you can just remove `--frozen-lockfile` here 🤔 and replace it with `npm ci`